tests/pubsub/integration/testfloodsub.nim

```diff
@@ -34,7 +34,7 @@ proc waitSub(sender, receiver: auto, key: string) {.async.} =
   let fsub = cast[FloodSub](sender)
   while not fsub.floodsub.hasKey(key) or
       not fsub.floodsub.hasPeerId(key, receiver.peerInfo.peerId):
-    await sleepAsync(100.millis)
+    await sleepAsync(50.millis)
     dec ceil
```

Comment:
`ceil` value should be increased two times, since now it will be reduce two times faster?

Reply:
could `checkUntilTimeout` / `checkUntilTimeoutCustom` be used here?

Reply:
I've reverted those `sleep` changes, so no need for further changes.